src/OrchardCore.Modules/OrchardCore.Themes/Assets.json

```diff
@@ -7,7 +7,21 @@
   },
   {
     "inputs": [
-      "Assets/js/theme-manager.js"
+      "Assets/js/constants.js",
+      "Assets/js/theme-loader.js"
+    ],
+    "watch": [
+      "Assets/js/constants.js",
+      "Assets/js/theme-loader.js"
+    ],
+    "output": "wwwroot/Scripts/theme-head.js"
+  },
+  {
+    "inputs": [
+      "Assets/js/theme-toggler.js"
+    ],
+    "watch": [
+      "Assets/js/theme-toggler.js"
     ],
     "output": "wwwroot/Scripts/theme-manager.js"
```

Comment:
In this case where you separate the theme-head from the toggler, then I think `theme-toggler.js` will be more clear, right?

Reply:
After cleanup the dependency, I thought theme-manager is better fit. Because I added dependency for the theme manager to the theme-head. So including the theme-manager will include the theme-head automatically. The reason I included them separately to ensure that the theme-head is loaded early on. So theme-manager would be fine.